main.js

```diff
@@ -0,0 +1,64 @@
+// Exercise 1:
+// Way 1:
+
+const isEqual = (arr1, arr2) => {
+  return (JSON.stringify(arr1) === JSON.stringify(arr2)) ? true : false
+}
+
+// Way 2:
+
+const isEqual2 = (arr1, arr2) => {
+  return (arr1.toString().replaceAll(',', '') === arr2.toString().replaceAll(',', '')) ? true : false
+}
+
+// Exercise 2:
+
+const flatten = arr => {
+  const convertToString = arr.toString().replaceAll(',', '')
+  return convertToString.split('')
+}
+
+// Exercise 3:
+
+const divideArray = (arr,chunk_size) => {
```

Comment:
```suggestion
const divideArray = (arr, chunk_size) => {
```
1: dấu cách
2: Đặt tên ko theo camel -> 5k